global.json

```diff
@@ -1,6 +1,6 @@
 {
     "sdk": {
         "version": "6.0.202",
-        "rollForward": "latestFeature"
+        "rollForward": "major"
```

Comment:
Not sure we want this.

Reply:
Most people don't have a compatible SDK on their machine. Using `major`, you can open and build the solution correctly.